index.ts

```diff
@@ -0,0 +1,19 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const createPacket = (data: any): { data: any, id: string} => {
```

Comment:
why `any`? we want the data we send in to still be known on the type after we get the result

can we link the data to the return type somehow?